src/iuse_actor.h

```diff
@@ -455,16 +455,23 @@ class salvage_actor : public iuse_actor
 
         /** Materials it can cut */
         std::set<material_id> material_whitelist = {
+            material_id( "alien_resin" ),
+            material_id( "acidchitin" ),
+            material_id( "bone" ),
             material_id( "cotton" ),
-            material_id( "leather" ),
-            material_id( "fur" ),
+            material_id( "chitin" ),
```

Comment:
Line should go up one level if it is sorted A-Z ascending.

Reply:
Ooh, thanks!